src/Classes/ServiceAPI/MyRadio_User.php

```diff
@@ -1372,6 +1401,17 @@ public function getEditForm()
                         ['value' => 'o', 'text' => 'Other']
                     ]
                 ]))
+                ->addField(new MyRadioFormField('contract', MyRadioFormField::TYPE_CHECK, [
```

Comment:
no, the point was to save paper by letting them do it here
(coincidentally, we couldn't find any paper for the printer at the time)